pkg/controller/clusters/status_test.go

```diff
@@ -128,11 +293,19 @@ func TestUpdateControlPlaneInitializedCondition(t *testing.T) {
 			err := clusters.UpdateClusterStatusForControlPlane(ctx, client, cluster)
 			g.Expect(err).To(BeNil())
 
-			condition := conditions.Get(cluster, anywherev1.ControlPlaneInitializedCondition)
+			condition := conditions.Get(cluster, tt.wantCondition.Type)
 			g.Expect(condition).ToNot(BeNil())
 
+			wantMessage := tt.wantCondition.Message
+			message := condition.Message
+
+			// Clear these out for easier comparison
 			condition.LastTransitionTime = v1.Time{}
+			condition.Message = ""
```

Comment:
this is a bit hacky
can we just compare the fields we are interested in directly? it's more explicit and easier to understand

Reply:
Yeah, we can do that